Cargo.toml

```diff
@@ -37,11 +37,12 @@ ida = []
 idas = []
 kinsol = []
 # Feature `klu` requires suitesparse (see README).
-klu = ["pkg-config"]
+klu = ["suitesparse_sys"]
 nvecopenmp = []
 nvecpthreads = []
 
 [dependencies]
+suitesparse_sys = { git = "https://github.com/martinjrobins/suitesparse-sys", optional = true, features = ["build_vendor"] }
```

Comment:
This cannot be merged for two reasons:
- crate.io [does not allow packages to be published with dependencies on code published outside of crates.io itself](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#the-role-of-the-version-key);
-  `build_vendor` should not be imposed on the user — if the library is present on his system, he may want to use that version.

Reply:
I've published this crate now and removed build_vendor so it'll use the system suitesparse if possible